### PR TITLE
Added logic to Render.bat, so that it is working for my directory of LyX

### PR DESCRIPTION
**Pull request by flyfl**

### CODE REVIEW

**Comment by jGleitz** [comment]
Lgtm

**Comment by RomanLangrehr** [comment]
Lgtm

**Comment by flyfl** [comment]
I accidently put the meeting minutes into this pull request, too. Next time I create new branches. Sorry for any confusion.

**Comment by jGleitz** [comment]
@flyfl: I already was confused where they came from and thought I missed something. Sorry for being unalert when reviewing!

I’ll review them now.

File: doc/Meeting Minutes/2015-11-20.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Meeting 2015-11-20
+
+## Development Phases
+
+#### Phase Managers
+- Internal and external contact person
+- Leader of presentation (Anyway, everyone else should participate in the presentation (2-3 min per Person))
+
+#### Presentation
+- The presentation of single phases is holded within our management,
```
Comment (jGleitz) [inline doc/Meeting Minutes/2015-11-20.md]: “held”, not “holded”

File: doc/Meeting Minutes/2015-11-20.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+(consultation between different chairs)
+
+
+
+## Software Requirement Specification 
+
+### DEADLINE 27.Nov, (officially 29.Nov.)
+
+#### Must have and nice to have criteria
+- We should try to be realistic, what can be done
```
Comment (jGleitz) [inline doc/Meeting Minutes/2015-11-20.md]: “We should try to be realistic about what can be done”

File: doc/Meeting Minutes/2015-11-20.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+#### Additional
+- Diagrams in Latex. Vector graphics are prefered (decision of our team).
+
+
+
+## Byte counter
+- Idea to use byte counter on common trace API (unclear if this approach works better)
+- Our design does not depend on the usage of byte counting
+- [Michael Kumperberg's Dissertation](http://digbib.ubka.uni-karlsruhe.de/volltexte/documents/1751666)
```
Comment (jGleitz) [inline doc/Meeting Minutes/2015-11-20.md]: Where did you get that link from? Awesome!

File: doc/Meeting Minutes/2015-11-20.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+
+
+## GitHub Interaction
+- No answering in time means someone is abstaining
```
Comment (jGleitz) [inline doc/Meeting Minutes/2015-11-20.md]: “Not answering in time” …

**Comment by jGleitz** [comment]
@flyfl Thanks for the well written and detailed minutes. Could you adopt my above comments in a new Pull Request?

**Pull request merged**